README.md

```diff
@@ -31,11 +31,15 @@ Maintain Kubernetes resources in sync easily.
 
 ## :tada: Introduction
 
-Kahoy is a minimal and flexible tool to sync/deploy your Kubernetes resource **raw** manifests and a cluster.
+Did you ever wish for a tool that could deploy all your manifests to a Kubernetes cluster without much hassle? Without worrying about applying the manifests in a specific order with kubectl, or setting up a complex CI workflow or even installing a more advanced tool that you first need to learn and observe.
```

Comment:
Not sure about this one, I intended to frame the problem in a way that people could better imagine where Kahoy fits. I tried to do this with this parragraph and the next one, please take a look.

Reply:
The idea is very good, but I would make it smaller, I think is too long for an introduction, at least the first paragraph, the second (_You probably..._) is perfect.